src/legacy/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -128,19 +140,42 @@ class VisEditorVisualization extends Component {
       this.setState((prevState) => {
         const newHeight = prevState.height + (keyCode === keyCodes.UP ? -15 : 15);
         return {
-          height: Math.max(MIN_CHART_HEIGHT, newHeight)
+          height: Math.max(MIN_CHART_HEIGHT, newHeight),
         };
       });
     }
   }
 
+  hasShowPanelIntervalValue() {
+    const type = get(this.props, 'model.type', '');
+
+    return [
+      'metric',
+      'top_n',
+      'gauge',
+      'markdown'
+    ].includes(type);
```

Comment:
Why not including also table visualization? isn't the same behaviour applied to tables?

Reply:
Good questions, probably we can but in requirements I see only  'top_n', 'gauge' and  'markdown'

![image](https://user-images.githubusercontent.com/20072247/53647258-3244dc00-3c4e-11e9-89ea-cc9544f42bd5.png)

Reply:
I think I'll add

Reply:
Isn't table only doing a terms aggregation on whatever you select? Interval wouldn't make sense here